trampoline/Cargo.toml

```diff
@@ -1,3 +1,5 @@
+[workspace]
```

Comment:
do we need it to be workspace? sorry I'm not super expert at this details, I thought that package should be enough as we don't have multiple libraries inside the trampoline